alica_engine/src/engine/AlicaEngine.cpp

```diff
@@ -192,7 +192,7 @@ IAlicaWorldModel* AlicaEngine::getWorldModel() const
 
 void AlicaEngine::subscribe(std::function<void(const YAML::Node& config)> reloadFunction)
 {
-    _configChangeListenerCBs.push_back(reloadFunction);
+    _configChangeListener.subscribe(reloadFunction);
```

Comment:
i suppose AlicaEngine::subscribe would be deprecated?

Reply:
Yes comment added